### DISCUSSION TITLE
closed (invalid fourCC: [255 255 122 23])

### DISCUSSION THREAD

**Discussion by adriano-peniche** [bluenviron/mediamtx#2208] [Questions]
Hello, I can't connect a camera via RTMP, it gives a invalid fourCC error. I think it might be related to the camera itself, but can't tell for sure.

mediamtx.yml:
```
protocols: [tcp] 
hlsDisable: no
webrtcDisable: yes
rtmpDisable: no
paths:
  all:
```
Output:
```
2023/08/15 21:27:33 INF MediaMTX v1.0.0
2023/08/15 21:27:33 INF [RTSP] listener opened on :8554 (TCP)
2023/08/15 21:27:33 INF [RTMP] listener opened on :1935
2023/08/15 21:27:33 INF [HLS] listener opened on :8888
2023/08/15 21:27:33 INF [SRT] listener opened on :8890 (UDP)
2023/08/15 21:27:46 INF [RTMP] [conn xxx.xxx.xx.xxx:52638] opened
2023/08/15 21:27:47 INF [RTMP] [conn xxx.xxx.xx.xxx:52638] closed (invalid fourCC: [255 254 112 23])
```

tshark dump:
[dump.txt](https://github.com/bluenviron/mediamtx/files/12353279/dump.txt)

**Comment by aler9** [comment]
Hello, open an issue and follow instructions in order to provide data that allows to replicate the issue, including the camera model.
